content/_data/sponsors.js

```diff
@@ -0,0 +1,132 @@
+// https://opencollective.com/oddbird-open-source/members/all.json
+import eleventyFetch from '@11ty/eleventy-fetch';
+import { config } from 'dotenv';
+import { groupBy } from 'lodash-es';
+
+// eslint-disable-next-line no-process-env
+if (!process.env.GITHUB_KEY) {
+  // Load .env variables with dotenv
+  config();
+}
+// eslint-disable-next-line no-process-env
+const GITHUB_KEY = process.env.GITHUB_KEY;
+
+const FilteredProfiles = [
+  // if there are backers we need to exclude…
+];
+
+const TIERS = ['Great Horned Owl', 'Blue-Footed Booby', 'Common Loon'];
+
+const getDefaultAvatarUrl = (url) => {
+  const slug = url.split('/').at(-1);
+  return slug ? `https://images.opencollective.com/${slug}/avatar.png` : null;
+};
+
+const loadGithubSponsors = async () => {
+  if (!GITHUB_KEY) {
+    // eslint-disable-next-line no-console
+    console.error('Github sponsors not loaded- set GITHUB_KEY in .env.');
+    return new Promise((resolve) => resolve([]));
+  }
+  const url = 'https://api.github.com/graphql?';
+  const { data } = await eleventyFetch(url, {
+    type: 'json',
+    duration: '0s',
+    directory: '.cache/eleventy-fetch/',
+    dryRun: false,
+    fetchOptions: {
+      method: 'POST',
+      headers: {
+        Authorization: `Bearer ${GITHUB_KEY}`,
+      },
+      body: JSON.stringify({
+        query: `
+        {
+  organization(login: "oddbird") {
+    name
+    sponsorshipsAsMaintainer(first: 100, activeOnly: false) {
+      totalCount pageInfo { hasNextPage }
+      nodes {
+        sponsorEntity{
+          ... on User { name avatarUrl websiteUrl url }
+          ... on Organization { name avatarUrl websiteUrl url }
+        }
+      }
+    }
+  }
+}`,
+      }),
+    },
+  });
+  if (data?.organization.sponsorshipsAsMaintainer.pageInfo.hasNextPage) {
+    // eslint-disable-next-line no-console
+    console.error(
+      'Good news- we have over 100 GitHub sponsors and need to implement pagination.',
+    );
+  }
+  return data?.organization.sponsorshipsAsMaintainer.nodes.map(
+    ({ sponsorEntity }) => ({
+      name: sponsorEntity.name,
+      tier: 'Common Loon',
```

Comment:
Will this put everyone from GH in the same tier? The [sponsor page](https://github.com/sponsors/oddbird) says

> your avatar [will be] displayed before and larger than lower level sponsors on all [OddTools pages on our website](https://www.oddbird.net/tools/)

Reply:
Yeah, for now. I'm thinking it'll be simpler to revisit it once we actually have recurring sponsors so that we can code against actual data.

Reply:
Yeah, it's a bummer if we can't get tiers _or_ total contributed, but this is fine for now 👍

Reply:
I think it's likely that we will be able to, but given the uncertainty over naming and data types, if I try to guess at what's being returned now, I'm likely to be wrong and need to redo it anyways.

Reply:
I thought the `tiers` field looked promising, but it returned `null` for me and seems like that's a [known issue](https://github.com/antfu-collective/sponsorkit/issues/60) 😕